slam-ws/src/load_waypoints/scripts/load_waypoints_client.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
We probably shouldn't commit this client as part of the package, but we should add this code to the Wiki because it shows how to access the server nicely!